cmake/projects/OpenCV/hunter.cmake

```diff
@@ -375,6 +386,7 @@ hunter_cmake_args(
     OpenCV
     CMAKE_ARGS
         BUILD_ANDROID_EXAMPLES=OFF
+        BUILD_JAVA=OFF
```

Comment:
Seems do the trick for Android toolchain:
* https://travis-ci.org/ingenue/hunter/builds/423312320